packages/react-composites/src/composites/common/CaptionsBanner.tsx

```diff
@@ -73,19 +68,25 @@ export const CaptionsBanner = (props: { isMobile: boolean }): JSX.Element => {
         )
       }
       {
-        /* @conditional-compile-remove(close-captions) */
-        <div className={containerClassName}>
-          <Stack horizontalAlign="center">
-            <Stack.Item style={{ width: props.isMobile ? mobileViewBannerWidth : desktopViewBannerWidth }}>
-              <_CaptionsBanner {...captionsBannerProps} {...handlers} strings={captionsBannerStrings} />
-            </Stack.Item>
-          </Stack>
-          {!props.isMobile && captionsBannerProps.isCaptionsOn && (
-            <div className={floatingChildClassName}>
-              <CaptionsBannerMoreButton onCaptionsSettingsClick={onClickCaptionsSettings} />
-            </div>
-          )}
-        </div>
+        /* @conditional-compile-remove(close-captions) */ captionsBannerProps.isCaptionsOn && (
```

Comment:
Why do we add a check here? We still wants captions banner to show when captions is not started yet but user already clicked start captions (loading banner)